src/components/content-tab-assistant.tsx

```diff
@@ -341,26 +354,53 @@ export function ContentTabAssistant( { selectedSite }: ContentTabAssistantProps
 
 	const disabled = isOffline || ! isAuthenticated || ! userCanSendMessage;
 
+	const renderChatContent = () => {
+		if ( ! isAuthenticated ) {
+			return isOffline ? (
+				<OfflineModeView />
+			) : (
+				<UnauthenticatedView onAuthenticate={ authenticate } />
+			);
+		}
+
+		if ( isOffline ) {
+			return messages.length > 0 ? (
+				<>
+					<WelcomeComponent
+						onExampleClick={ handleSend }
+						showExamplePrompts={ messages.length === 0 }
+						messages={ welcomeMessages }
+						examplePrompts={ examplePrompts }
+					/>
+					<AuthenticatedView messages={ messages } isAssistantThinking={ isAssistantThinking } />
+					<OfflineModeView />
```

Comment:
It would be great if we append here `<div ref={ endOfMessagesRef } />` to automatically scroll to the end of the content when navigating to the assistant tab.